apps/explorer/src/app/routes/home/index.tsx

```diff
@@ -3,7 +3,6 @@ import { useDocumentTitle } from '../../hooks/use-document-title';
 
 const Home = () => {
   const classnames = 'mt-4 grid grid-cols-1 lg:grid-cols-2 lg:gap-4';
-
   useDocumentTitle();
 
   return (
```

Comment:
**nitpick (blocking)**:    (this might have been more appropriate for https://github.com/vegaprotocol/frontend-monorepo/pull/3254/files#diff-bc0a53cf4d28f6f08a2a88579a08623af3284c12760b874e6a4ba340fe2d49ac)
<img width="626" alt="Screenshot 2023-03-22 at 16 11 52" src="https://user-images.githubusercontent.com/6678/226968391-14d595c5-b157-4383-9554-e7b45ccface6.png">
Stats manager has a similar looking title section to the breadcrumbs, so this ends up looking weird. I see two choices:

1. No breadcrumbs on the homepage
2. Remove the title and only have the breadcrumbs

I vote number 2. It's not consistent, but then the page is very different to the others.